pkg/csi/networkBuilder.go

```diff
@@ -370,18 +371,27 @@ func createNetFromLayoutConfig(conf NetworkLayoutConfiguration) (*IPV4Network, e
 
 	if conf.GroupNetworksByCabinetType && conf.SubdivideByCabinet {
 		if strings.HasSuffix(conf.Template.Name, "RVR") {
-			tempNet.GenSubnets(conf.CabinetDetails, conf.CabinetCIDR, "river")
+			tempNet.GenSubnets(conf.CabinetDetails, conf.CabinetCIDR, CabinetClassFilter(sls_common.ClassRiver))
```

Comment:
I just found a bug in this area of the code. Need to fix it before merging

Reply:
VlanRanges are now being correctly calculated. The filter has been corrected to correctly filter for both River and EX2500 cabinets with air cooled chassis.
```
cat surtur/sls_input_file.json| jq .Networks.HMN_RVR.ExtraProperties.VlanRange
[
  1513,
  1513
]
```